### PR TITLE
sslscan: update to 2.0.10

### PR DESCRIPTION
**Pull request by herbygillot**
###### Tested on

macOS 10.15.7 19H1217 x86_64
Xcode 12.4 12D4e

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -d install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: net/sslscan/Portfile [hunk 1]
```diff
@@ -35,3 +35,5 @@ build.args      PREFIX="${prefix}" \
                 LDFLAGS="[get_canonical_archflags ld] -L${prefix}/lib"
 
 destroot.args   PREFIX="${prefix}"
+
+github.tarball_from archive
```
Comment (raimue) [inline net/sslscan/Portfile:39]: Hm, what is the reason for this change? The default 'tarball' seems fine to me?

Comment (herbygillot) [reply net/sslscan/Portfile:39]: The `tarball` URL is actually legacy, and though Github still supports it, it's considered deprecated.  So as projects are updated, we convert them to use `archive` or `release` URLs instead of the default.  Then perhaps at some point, the `github` portgroup's `tarball_from`option can be made to default to `archive`.

More discussion here:
- https://github.com/macports/macports-ports/pull/2587
- https://trac.macports.org/ticket/40518

Comment (raimue) [reply net/sslscan/Portfile:39]: This is still the default in the port group and it works for many ports.

If we want to get rid of `tarball`, I would rather change it in the port group. If there are concerns that it could break ports, then make a github-1.1 port group with the changed default and update the ports. But setting this individually in each port kind of goes against the purpose of a port group in my opinion.

Comment (herbygillot) [reply net/sslscan/Portfile]: Alright, removed.

**Comment by raimue** [comment]
Thank you! 👍

**Comment by herbygillot** [comment]
Thanks @raimue
